### PR TITLE
Default text for user profile

### LINKED ISSUE #108: Default text for user profile
Display `User has no posts` when len(post)=0

### PR DESCRIPTION
**Pull request by amalaabraham**
Resolves #108 

- Display user has no posts when there are no posts.
- User can create a post if he wants to.

Screenshot (88)

### CODE REVIEW

File: frontend/src/Components/UserProfile.js [hunk 1]
```diff
@@ -67,15 +68,22 @@ export default function UserProfile({ id }) {
...
...87,937.38805,446.09752,940.57354,446.16634Z" transform="translate(-178.21027 -129.74161)" fill="#2f2e41"/><path d="M971.04643,580.81327l4.35678,165.33964a4,4,0,0,1-4.20243,4.10017l-14.35213-.73225a4,4,0,0,1-3.74658-3.36683L936.031,638.76494a2,2,0,0,0-3.92705-.12222l-23.10758,103.3966a4,4,0,0,1-5.24374,2.89644l-14.24374-.87a4,4,0,0,1-2.64986-4.05306l12.434-160.76415Z" transform="translate(-178.21027 -129.74161)" fill="#2f2e41"/><path d="M978.03835,502.28255c2.403-25.05537-16.01923-47.71792-41.12634-49.5007-10.81772-.76813-27.61219-1.04273-33.14841,13.13648-16.6774,42.71375,12.51774,41.67582,2.45884,78.68292s-18.94294,39.128-5.14121,41.80029,75.86671,18.447,73.31817-7.60005C972.83287,562.79095,975.64139,527.27536,978.03835,502.28255Z" transform="translate(-178.21027 -129.74161)" fill="#475177"/><path d="M908.71686,461.57372l0,0a12.08366,12.08366,0,0,1,16.29393,13.1781l-7.8391,48.991a47.876,47.876,0,0,1-10.26518,22.80677L881.44705,577.5737a4,4,0,0,1-6.06458.13914L869.335,570.997a4,4,0,0,1-.196-5.11813l1.78109-2.31142a134.7031,134.7031,0,0,0,27.04467-66.179l3.11643-25.98473A12.08366,12.08366,0,0,1,908.71686,461.57372Z" transform="translate(-178.21027 -129.74161)" fill="#475177"/></svg>
```
Comment (aitikgupta) [inline frontend/src/Components/UserProfile.js]: This definitely needs some linting (which isn't running atm).

File: frontend/src/Components/UserProfile.js [hunk 2]
```diff
@@ -67,15 +68,22 @@ export default function UserProfile({ id }) {
...
+              {userProfile.user_id === user_id ? (
+                <h1 className="text-2xl">No posts to show.<A href={`/home`} className="hover:text-blue-500">Start creating your posts here.</A></h1>
+              ):<h1 className="text-2xl">No posts to show.</h1>}
```
Comment (aitikgupta) [inline frontend/src/Components/UserProfile.js]: 1. Move the second sentence to a new line?
2. It's not obvious that's it's a link (so just removed the "hover:" part, don't know if that will work)
image

```suggestion
                <h1 className="text-2xl">No posts to show.
                <A href={`/home`} className="text-blue-500">Start creating your posts here.</A></h1>
              ):<h1 className="text-2xl">No posts to show.</h1>}     
```

**Review by aitikgupta — COMMENTED** [review]
Deployed URL: https://607844cc3903f000089787dc--sociomark.netlify.app/

This looks so good @amalaabraham! 🎉 
Just a few nitpicks since we don't have our linters running :(

**Comment by amalaabraham** [comment]
@aitikgupta Linting is done :) Also, what I meant is when you click on start creating your posts, it goes to the home page so that the user can add his first post.  And this feature is only applicable to the post owner only. When the same user goes to some other's profile, if there are no posts, only "no posts to show" is displayed.

**Comment by aitikgupta** [comment]
@amalaabraham yeah I understand that, the suggestions I made were just related to styling, the logic you've implemented is indeed correct.

**Comment by amalaabraham** [comment]
@aitikgupta Cool! :)

**Review by bodhisha — APPROVED** [review]
Looks Good!! 🎉 @amalaabraham

File: frontend/src/Components/UserProfile.js (hunk 2 above)
Comment (aitikgupta) [reply frontend/src/Components/UserProfile.js]: A rebase is required after this @amalaabraham

Comment (amalaabraham) [reply frontend/src/Components/UserProfile.js]: @aitikgupta Updated the style! :) Rebased too.

Screenshot (91)

**Comment by aitikgupta** [comment]
@amalaabraham  File:[frontend/src/Components/UserProfile.js] > You need to run Prettier on this file

**Comment by aitikgupta** [comment]
@amalaabraham thanks for your contribution! (and for sticking around with those painful linters)

**Pull request merged**